pandas/core/algorithms.py

```diff
@@ -1155,7 +1155,7 @@ def compute(self, method):
         n = min(n, narr)
 
         kth_val = algos.kth_smallest(arr.copy(), n - 1)
-        ns, = np.nonzero(arr <= kth_val)
+        (ns,) = np.nonzero(arr <= kth_val)
```

Comment:
why this change?

Reply:
Sorry It was changed automatically by black pandas, because I didn't run the right version of the latter.